.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
             "args": ["--sync", "all", "--debugpath", "${input:debugPath}", "--debugbranch", "${input:debugBranch}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Push Binaries",
             "args": ["--push", "Binaries.zip", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Binaries",
             "args": ["--sync", "binaries", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Engine Build",
             "args": ["--sync", "engine", "--bundle", "${input:bundleSelection}", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Generate DDC",
             "args": ["--sync", "ddc", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Update Engine Version to Latest",
```

Comment:
While running the configuration

```
Exception has occurred: SystemExit
1
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 171, in sync_handler
    sys.exit(1)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 338, in main
    sync_handler(args.sync, args.repository, args.bundle)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 357, in <module>
    main()
```
Log:
`11-May-20 12:53:18 [ERROR]  Error while trying to fetch latest engine version`
```

Reply:
Should be fixed now